modules/tribes/client/components/JoinButton.component.js

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+import LeaveTribeModal from './LeaveTribeModal';
+import { OverlayTrigger, Tooltip } from 'react-bootstrap';
+
+import * as api from '../api/tribes.api';
+
+function JoinButtonPresentational({ isMember, isLoading, joinLabel='Join', joinedLabel='Joined', tribe, isLogged, onToggle }) {
+  const { t } = useTranslation('tribes');
+
+  const ariaLabel = (isMember) ? t('Leave Tribe') : t(`${joinLabel} ({{label}})`, { label: tribe.label });
+  const title = (isMember) ? t(joinedLabel) : t(joinLabel);
+
+  if (!isLogged) {
```

Comment:
`isLoggedIn` seems clearer to me, `isLogged` sounds like it's something related to login to my brain.

Reply:
Done in c5c33f984a7cdb75721ceed50cd222bc2b55b327